build/docker/tools/tke-installer/Dockerfile

```diff
@@ -29,9 +29,16 @@ RUN apk add --no-cache \
     curl \
     tcpdump \
     docker \
-    ansible
+    ansible \
+    containerd
 
 RUN apk --update-cache --repository http://dl-3.alpinelinux.org/alpine/edge/testing/ --allow-untrusted add lrzsz
+RUN wget https://github.com/containerd/nerdctl/releases/download/v0.10.0/nerdctl-0.10.0-linux-amd64.tar.gz \
```

Comment:
For arm64 tke-installer, these bin will not work.

Reply:
arch=arm64 version=v1.7.0 && wget https://tke-release-1251707795.cos.ap-guangzhou.myqcloud.com/tke-installer-linux-$arch-$version.run{,.sha256} && sha256sum --check --status tke-installer-linux-$arch-$version.run.sha256 && chmod +x tke-installer-linux-$arch-$version.run && ./tke-installer-linux-$arch-$version.run

Reply:
Add arm packages.